cgo/blockstore.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	blocks "github.com/ipfs/go-block-format"
 	"github.com/ipfs/go-cid"
-	blockstore "github.com/ipfs/go-ipfs-blockstore"
+	"github.com/ipfs/go-ipld-format"
```

Comment:
Can we import this as `ipld`? I think it looks clearer that way (see, eg. [here](https://github.com/ipfs/go-ipfs-blockstore/pull/48/files#diff-d0909a447e05cb276100d4f9dd10ffcab536acd3d91d84111361df2188c4e186R163))

Reply:
@masih once addressed plz feel free to land!